docs/developer_guide/index.rst

```diff
@@ -8,35 +8,35 @@ You will need conda/mamba and CUDA installed, see the :ref:`Installation` instru
 
 *Note that the Python commands given below are for Linux only. To run the commands on Windows, replace* :code:`python3` *with* :code:`python`.
 
-First download the Mantid Imaging source code using Git.
+First download the Mantid Imaging source code using Git::
 
-:code:`git clone https://github.com/mantidproject/mantidimaging.git`
+    git clone https://github.com/mantidproject/mantidimaging.git
 
-If you have a github account you can use ssh access:
+If you have a github account you can use ssh access::
```

Comment:
We should probably mention that an ssh key is required for installing with ssh and also include instruction for cloning using the GitHub CLI if we're touching this part of the docs anyway

Reply:
I've added a link to the github ssh docs. I've never used the github cli  tools.

Reply:
Recommend giving it a try. it's really nice